docs/user/rstudio/ide/guide/data/data-viewer.qmd

```diff
@@ -0,0 +1,131 @@
+---
+title: "Data Viewer"
+---
+
+RStudio includes a Data Viewer that allows you to look inside data frames and other rectangular data structures. The viewer also allows includes some simple exploratory data analysis (EDA) features that can help you understand the data as you manipulate it with R or Python.
+
+## Starting the viewer
+
+You can invoke the viewer in a console by calling the `View` function on the data frame you want to look at. 
+
+For example, to view the `mpg` dataset from `ggplot2`, execute these commands in the R console:
+
+``` r
+library(ggplot2)
+mpg_df <- ggplot2::mpg[c("manufacturer", "model", "displ", "cty", "hwy")]
+View(mpg_df)
+```
+
+![](images/mpg_df-view.png){fig-alt="A datatable in the Data Viewer displaying 5 columns and 10 rows of the mpg_df dataset."}
+
```

Comment:
is the red box on this image intentional?
![image](https://user-images.githubusercontent.com/114421526/197590372-b001e0f7-9e96-40c4-acce-491dfcb83348.png)

Reply:
Yikes - corrected!